spec/features/super_admin/allowlists/organisation_names/delete_custom_organisation_spec.rb

```diff
@@ -16,15 +16,15 @@
     end
 
     it "removes the correct organisation name from the list" do
-      click_on "Yes, remove this organisation"
+      click_on "Yes, remove DummyOrg1 from the allow list"
```

Comment:
I think we should have a test here that would have failed before your fix and passes after.